api/agent.go

```diff
@@ -16,6 +18,19 @@ type Agent struct {
 	region     string
 }
 
+// KeyringResponse is a unified key response and can be used for install,
+// remove, use, as well as listing key queries.
+type KeyringResponse struct {
```

Comment:
Well that explains it! Thanks. (Wish I could mark comments as _addressed_ in this new PR mode without full on deleting them...)